tests/FSharpAux.Mocha.Tests/fable/src/FSharpAux.Core/Dict.js

```diff
@@ -0,0 +1,183 @@
+import { contains, toList as toList_1, toArray as toArray_1, map } from "../../fable_modules/fable-library.4.0.6/Seq.js";
```

Comment:
Also, i am pretty sure that these imports won't work anyways without running fable after checking out the repo locally

Reply:
you are right, didn't check in depth and just noticed that the fable folder did not get commit.

Reply:
Adjusted